src/analysis/Environments/Environments.ts

```diff
@@ -280,10 +280,14 @@ export const Environments = (props: EnvironmentsProps): ReactNode => {
       includeLabels: 'saturnApplication,saturnWorkspaceNamespace,saturnWorkspaceName',
     };
 
+    const noCromwellAppTypes = ['GALAXY', 'HAIL_BATCH', 'WDS'];
```

Comment:
I think it would be more maintainable to reverse the logic, and list the cromwell app types here and then exclude them when doing the filtering below